config/manifests/kustomization.yaml

```diff
@@ -6,3 +6,19 @@ resources:
 - ../metallb_rbac
 - ../samples
 - ../scorecard
+
+# These patches remove the unnecessary "cert" volume and its manager container volumeMount.
+patchesJson6902:
+- target:
+    group: apps
+    version: v1
+    kind: Deployment
+    name: controller-manager
+    namespace: system
+  patch: |-
+    # Remove the manager container's "cert" volumeMount, since OLM will create and mount a set of certs.
+    - op: remove
+      path: /spec/template/spec/containers/0/volumeMounts/0
```

Comment:
the memcache file had comment about update indicies and used different path from the above just curious why yours is different ?

Reply:
i did the update and removed the comment. we only have one container: https://github.com/metallb/metallb-operator/blob/c6c7d50278d6013d1359189e069157e890ae3fc4/config/webhook-with-certmanager/manager_webhook_patch.yaml#L10

Reply:
/lgtm2